<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>简易FFT演示程序v0.4</string>
  </property>
  <layout class="QHBoxLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <widget class="QPushButton" name="plotBtn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>340</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>绘图</string>
      </property>
     </widget>
     <widget class="QPushButton" name="save_figure_Button">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>340</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>保存图片</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>函数式</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>320</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>FFT点数</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="funcexp">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.5*sin(2*pi*50*x)+0.5*sin(2*pi*30*x)</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="data1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>211</width>
        <height>181</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'宋体'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;[ 1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1.]&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>数组</string>
      </property>
     </widget>
     <widget class="MPL_Widget" name="plotWidget" native="true">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>0</y>
        <width>361</width>
        <height>381</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>360</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>频率(omega/π)</string>
       </property>
      </widget>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>320</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>256</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>512</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1024</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2048</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>采样频率(Hz)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>采样点数</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="samplefreq">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>1000</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="samplepoint">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>512</string>
      </property>
     </widget>
     <widget class="QPushButton" name="getdata_Button">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>获取数组</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>280</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>度</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>250</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>幅</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sample_period_time">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.001</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>采样间隔时间(s)</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MPL_Widget</class>
   <extends>QWidget</extends>
   <header>mpl_pyqt4_widget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
